### PR TITLE
Add REST API method to list online users in a room

### PR DESCRIPTION
**Pull request by wtsarchive**
@RocketChat/core 

This pull request re-implements the API method to get online users in a room. It was already implemented by https://github.com/RocketChat/Rocket.Chat/pull/2918 but it got lost when moving the api to /v1 namespace.

### CODE REVIEW

File: packages/rocketchat-api/server/v1/channels.js [hunk 1]
```diff
@@ -370,6 +370,34 @@ RocketChat.API.v1.addRoute('channels.list.joined', { authRequired: true }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('channels.online', { authRequired: true }, {
+	get: function() {
+		const room = findChannelById(this.queryParams.roomId);
+
+		if (room.statusCode) {
```
Comment (graywolf336) [inline packages/rocketchat-api/server/v1/channels.js]: Can you merge the latest code? As this line of code isn't needed any more.

File: packages/rocketchat-api/server/v1/channels.js [hunk 2]
```diff
@@ -370,6 +370,34 @@ RocketChat.API.v1.addRoute('channels.list.joined', { authRequired: true }, {
 	}
 });
 
+RocketChat.API.v1.addRoute('channels.online', { authRequired: true }, {
```
Comment (graywolf336) [inline packages/rocketchat-api/server/v1/channels.js]: Can you add this to the `groups` as well? And if you're up for it, create the documentation on this over on our docs? https://github.com/RocketChat/Rocket.Chat.Docs/tree/master/6.%20Developer%20Guides/REST%20API

**Comment by engelgabriel** [comment]
@graywolf336 can you do the sync so we can merge this?

**Comment by rodrigok** [comment]
@graywolf336 Can you review this again?

**Comment by graywolf336** [comment]
@rodrigok What would you think about changing it from just `username` to include the user's id as well?

File: packages/rocketchat-api/server/v1/channels.js (hunk 2 above)
Comment (rodrigok) [inline packages/rocketchat-api/server/v1/channels.js]: Done

**Comment by rodrigok** [comment]
@graywolf336 Done :)

**Pull request merged**